docker/Dockerfile.binary

```diff
@@ -15,4 +15,9 @@ FROM docker.io/debian:bullseye-slim
 RUN apt-get update && apt-get install -y ca-certificates tini && apt-get clean
 COPY --from=cargo-build /src/target/release/dshackle_health_adapter /usr/local/bin/dshackle_health_adapter
 
-ENTRYPOINT ["/usr/bin/tini"]
+ENV HEALTH_URL=""
+ENV NODE_ID=""
+ENV BIND_ADDRESS=""
+ENV UNHEALTHY_LAG=""
+
+ENTRYPOINT /usr/local/bin/dshackle_health_adapter --bind-address $BIND_ADDRESS --health-url $HEALTH_URL --node-id $NODE_ID --unhealthy-lag $UNHEALTHY_LAG
```

Comment:
Is this intended? We added `env` to the `clap` derive, so it should automatically pick up environment variables if they are specified.

Reply:
Oh, I wasn't aware of the `clap` derive. I'll test it.